base/build.gradle

```diff
@@ -50,9 +55,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
     // UI
     implementation "androidx.core:core-ktx:$rootProject.coreKtxVersion"
-    implementation "androidx.percentlayout:percentlayout:$rootProject.androidxPercentlayoutVersion"
-    implementation "androidx.recyclerview:recyclerview:$rootProject.androidxRecyclerviewVersion"
-    implementation "com.google.android.gms:play-services-wearable:$rootProject.googlePlayServicesClientVersion"
+
     implementation "com.google.android.support:wearable:$rootProject.wearableSupportLibraryVersion"
```

Comment:
Oh this is interesting, do we want `base` to use the support library still?

Reply:
No, well, I wanted to get sign-off on the complete module before I carried over the same code to the base module... now that you signed off on that, I added the fixes for base to the PR so it should be 100% ready for approval. 

I will send you doc with code lab as well. Thanks!